### ISSUE TITLE
Numpy Version Mismatch Issue With Onnxruntime module

### ISSUE THREAD

**Issue by tonmoy50** [deepinsight/insightface#2474]
The main issue is with the `draw_on` function of `FaceAnalysis` module. It is using `np.int` for type casting but this has been deprecated and removed from numpy starting 1.20 if I am not mistaken. I think it need to be updated as I cannot downgrade numpy because the Onnxruntime module depends on numpy version >1.21.

For reference, I am using 
```
Onnxruntime=1.16.2
Numpy=1.26.2
Insightface=0.7.3
```

**Comment by nttstar** [comment]
Yes, it has been fixed in the source code, but it hasn't been pushed to the pip package yet. I will complete the upgrade after some days.

**Comment by tonmoy50** [comment]
Great! Is it okay to leave the issue open until then?

**Comment by jens-markussen** [comment]
Any update on this? I ran into exactly the problem described about (np.int error - not just warning).

**Comment by allen0125** [comment]
Any update on this?

**Comment by tonmoy50** [comment]
I don't think they made any changes. I cloned their repository and manually made the changes to work for me

**Comment by satishjasthi** [comment]
duplicate of #2404
